client/src/lib/error-telemetry.js

```diff
@@ -9,22 +9,47 @@ const Tail   = require('tail').Tail;
 const fs     = require('fs');
 const logger = require('winston');
 
-const TARGET = '/tmp/test';
+/**
+ * Default behaviour for the output where to send data to when the watched logging file has
+ * a modification detected.
+ */
+function callErrorTelemetryService(app, text) {
+
+  const api = app.service('errorTelemetry');
+
+  api.create({
+    log: text // FIXME: not very happy with this, to design in JEP: json in (json) log field?
+  }).then((res) => {
+    logger.info('pushed as '+ res.id);
+  }).catch((res) => {
+    logger.error('Failed to push log:', res);
+  });
+}
 
 class ErrorTelemetry {
   constructor(app, options) {
     this.app = app;
     this.options = options;
   }
 
-  setup() {
+  /**
+   * monitoredFile: path to the log file to watch
+   * outputFunction(app,line): the function that will be called on each new line detected
+   */
+  setup(monitoredFile, outputFunction=callErrorTelemetryService) {
     logger.info('Setting up error logging...');
-    const loggingFile = this.fileToWatch();
+
+    let loggingFile = '';
+    if(monitoredFile) {
```

Comment:
I'll need to find an eslint rule that will break on the lack of spaces between reserved words and parenthesis :smiling_imp: